### PR TITLE
Fixes #6406: merge all function may not checkout branch correctly

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6406

### CODE REVIEW

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -728,6 +732,7 @@ def merge(old, new):
 def merge_to_next(old):
   new = None
   for idx, branch in enumerate(ALL_BRANCHES):
+    print("idx="+str(idx) + " branch=" + branch)
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: Is it wanted ? or a debug output ?

Comment (peckpeck) [inline scripts/rudder-dev/rudder-dev]: Oops sorry

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
@VinceMacBuche ping

**Comment by VinceMacBuche** [comment]
Awesome, great, thanks \o/

**Pull request merged**